### PR TITLE
Add support for the Person resource

### PR DESCRIPTION
**Pull request by remi-stripe**
cc @stripe/api-libraries

### CODE REVIEW

File: test/resources/Persons.spec.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+// Use spy from existing resource:
+person._request = stripe.customers._request;
+
+describe('Person Resource', function() {
+  describe('create', function() {
+    it('Sends the correct request', function() {
+      person.create({
+        first_name: "John",
+      });
```
Comment (remi-stripe) [inline test/resources/Persons.spec.js]: This part makes no sense to me. I know it works and why but that feels like a really weird way to approach this test and I'm ~confused why we even allow this.

Comment (jlomas-stripe) [reply test/resources/Persons.spec.js]: What part makes no sense?

Comment (remi-stripe) [reply test/resources/Persons.spec.js]: The part where to create a brand new object (which means said object does not exist yet) you use a pre-filled fake object.

File: test/resources/Account.spec.js [hunk 2]
```diff
@@ -231,4 +231,116 @@ describe('Account Resource', function() {
...
+        stripe.account.updatePerson('acct_123', 'person_123', {
+          first_name: 'John',
+        });
+        expect(stripe.LAST_REQUEST).to.deep.equal({
+          method: 'POST',
+          url: '/v1/accounts/acct_123/persons/person_123',
+          headers: {},
+          data: {first_name: 'John'},
+        });
+      });
```
Comment (ob-stripe) [inline test/resources/Account.spec.js]: Missing a test for update + specified auth.

**Comment by remi-stripe** [comment]
@ob-stripe PTAL

**Comment by ob-stripe** [comment]
Released as 6.13.0.

**Pull request merged**